source/reference/minio-mc/mc-version-enable.rst

```diff
@@ -0,0 +1,126 @@
+.. _minio-mc-version-enable:
+
+=====================
+``mc version enable``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc version enable
+
+
+Syntax
+------
+
+.. start-mc-version-enable-desc
+
+The :mc:`mc version enable` command enables versioning on the specified bucket.
+
+.. end-mc-version-enable-desc
+
+Mutually exclusive with :mc-cmd:`~mc version suspend` and :mc-cmd:`~mc version info`.
```

Comment:
Since these are CLI "top" commands we can safely drop this sentence

Reply:
also removed the other similar lines